internal/storage/rocksdb/store.go

```diff
@@ -380,6 +381,17 @@ func (rdb *rocksDB) CompareAndSet(key, expect, update []byte) (bool, error) {
 	return err == nil, err
 }
 
+func (rdb *rocksDB) GetKeySpaceSize() (int64, error) {
+	defer rdb.opts.statsCli.Timing("rocksdb.property.latency.dbSize", time.Now())
+	keySize := rdb.db.GetProperty("rocksdb.estimate-num-keys")
```

Comment:
This might be incorrect, Need  to sum up sizes for multiple column families. Add test case to verify all scenarios